pom.xml

```diff
@@ -26,7 +26,7 @@
             of the SDK generator used to generate your SDK code.
             See this link for details: https://github.ibm.com/CloudEngineering/openapi-sdkgen/wiki/Compatibility-Chart
         -->
-        <sdk-core-version>9.5.4</sdk-core-version>
+        <sdk-core-version>9.9.0</sdk-core-version>
```

Comment:
We're up to 9.9.1 now :)

Reply:
Yep, thanks - 9.9.0 matches the generator version used here though.